src/compiler/checker.ts

```diff
@@ -19672,6 +19710,12 @@ namespace ts {
                     case SyntaxKind.ExpressionWithTypeArguments:
                     case SyntaxKind.HeritageClause:
                         return false;
+                    case SyntaxKind.ExpressionWithTypeArguments:

```

Comment:
linter is failing right here because I think the merge ended up with duplicate lines eg on 19710 and 19713

Reply:
_TypeScript linters, who needs 'em?_